### PR TITLE
add OpenLibrary data seed

### PR DESCRIPTION
**Pull request by maxlath**

### CODE REVIEW

File: tests/integration/data/get_openlibrary_entry_from_isbn.test.js [hunk 1]
```diff
@@ -11,4 +11,25 @@ describe('get_openlibrary_entry_from_isbn', () => {
     entry.works[0].claims['wdt:P648'].should.equal('OL26437584W')
     entry.authors[0].claims['wdt:P648'].should.equal('OL9962491A')
   })
+
+  it('should find entries with only ISBN-10 from an ISBN-13', async () => {
+    const entry = await getOpenLibraryEntryFromIsbn('978-2-7152-1841-3')
+    entry.edition.claims['wdt:P648'].should.equal('OL914930M')
+    entry.edition.claims['wdt:P244'].should.equal('95211094')
+  })
```
Comment (jum-s) [inline tests/integration/data/get_openlibrary_entry_from_isbn.test.js:19]: all those tests are red for me
`TypeError: data.key.split(...).at is not a function`

**Review by jum-s — CHANGES_REQUESTED** [review]
even though node 16 should accept `.at` i still have a ` TypeError: data.key.split(...).at is not a function` on all the tests added in this PR.

**Review by jum-s — APPROVED** [review]

**Comment by maxlath** [comment]
@jum-s `>= 16.6.0`? Could that be that your command to run tests (via docker?) relies on an older version of Node than the server?

**Comment by jum-s** [comment]
all good

**Pull request merged**